src/main/java/org/folio/auth/authtokenmodule/apis/FilterApi.java

```diff
@@ -63,20 +63,18 @@ public FilterApi() {}
    * @param routeApi A reference to the RouteApi object, which this API depends on for route
```

Comment:
Don't forget to add a new `@param` here for your new param

Reply:
done